.github/workflows/codespell.yml

```
@@ -0,0 +1,22 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+permissions:
+  contents: read
```

According to https://github.com/codespell-project/codespell/blob/master/.github/workflows/codespell.yml#L6, the `codespell-project/actions-codespell` action doesn't need any permissions.
```suggestion
permissions: {}
```